app/controllers/Api.scala

```diff
@@ -58,8 +56,8 @@ object Api extends Controller with PanDomainAuthActions {
     }
 
     CommonAPI.getStubs(qs).asFuture.map {
-      case Left(err) => InternalServerError
-      case Right(contentResponse) => Ok(Json.toJson(contentResponse))
+      case Left(_) => InternalServerError
+      case Right(contentResponse) => Ok(contentResponse.asJson.spaces4)
```

Comment:
Should this maybe just be without pretty spaces? Since the app/browser can handle that and it might slightly reduce the response size? I don't know if it matters hugely though.

Reply:
Good point. I got too excited about that 😆 I'll change it!